### PR TITLE
[NOT a pull request] Another attempt at modelling a TRS implementation

### PR DESCRIPTION
**Pull request by jadelkhoury**
This is a second attempt at illustrating how (inMemory) TRS support can look like in LyoDesigner.

The previous example had other irrelevant code changes. (https://github.com/OSLC/lyo-adaptor-sample-modelling/pull/46)

Plus, I think I fixed one main comment from @berezovskyi .

### CODE REVIEW

**Comment by jadelkhoury** [comment]
@berezovskyi what do you think now?
You can ignore the first of the 3 commits. It is not relevant for the TRS.

File: adaptor-rm-webapp/src/main/java/com/sample/rm/RMToolManager.java [hunk 1]
```diff
@@ -115,6 +128,16 @@ public static void contextInitializeServletListener(final ServletContextEvent se
...
         // Start of user code contextInitializeServletListener
         initializeRequirements(37);
+        
+
+        ArrayList<URI> uris = new ArrayList<URI>();
+        ArrayList<Requirement> requirements = getRequirements();
+        for (Iterator iterator = requirements.iterator(); iterator.hasNext();) {
+            uris.add(((Requirement) iterator.next()).getAbout());
+        }
+        PagedTrs temp = new PagedTrsFactory().getInmemPagedTrs(50, 50, uris);
```
Comment (berezovskyi) [inline adaptor-rm-webapp/src/main/java/com/sample/rm/RMToolManager.java:138]: `InmemPagedTrs temp`

File: adaptor-rm-webapp/src/main/java/com/sample/rm/RMToolManager.java [hunk 2]
```diff
@@ -115,6 +128,16 @@ public static void contextInitializeServletListener(final ServletContextEvent se
...
+        pagedTrs = temp;
+        trsEventHandler = (TrsEventHandler) temp;
```
Comment (berezovskyi) [inline adaptor-rm-webapp/src/main/java/com/sample/rm/RMToolManager.java:140]: and remove the type cast

**Comment by jadelkhoury** [comment]
Resolved your last 2 comments. Good enough to generate then?

**Comment by berezovskyi** [comment]
yes, let's do it!

**Comment by jadelkhoury** [comment]
The LyoDesigner patch is now merged. This branch is no longer relevant.

**Pull request closed without merging**